game.c

```diff
@@ -2,32 +2,60 @@
 #include <stdlib.h>
 #include <ncurses.h>
 
+#define WALL '#'
```

Comment:
I'm not sure how best to handle display configurations like this. I think a pile of constants isn't the right thing, but it's better than hardcoded strings so that's a start. I suspect that the real (non-game design) complexity in a project like this will always be around configuring the mobs and tiles.

Reply:
Even I thought that this was going to be a problem and you know my style of programming! I would put it in an #include?